### ISSUE TITLE
zymo reference

### ISSUE THREAD

**Issue by hasindu2008** [Kirk3gaard/2023-basecalling-benchmarks#2]
Thanks (again) for this 5KHz data set, which is being very useful in testing our POD5<->S/BLOW5 converter https://github.com/Psy-Fer/blue-crab, 
Are these the correct zymo references that we should use with this dataset?
https://s3.amazonaws.com/zymo-files/BioPool/ZymoBIOMICS.STD.refseq.v2.zip

https://files.zymoresearch.com/protocols/_d6300_zymobiomics_microbial_community_standard.pdf

**Comment by Kirk3gaard** [comment]
Hi @hasindu2008 
They are still the newest released ones yes. However, I have used the pacbio hifi based ones that zymo sent me personally but for some reason have not made public yet. I have asked them several times about updating their references.
Best regards
Rasmus

**Comment by hasindu2008** [comment]
The identity scores against the one I linked above were pretty good, so I guess they are quite close :D

**Comment by Kirk3gaard** [comment]
Yes they are pretty good but ONT data is now sufficiently good that even tiny errors matter for benchmarking 😃
